docs: Update Create Cluster docs with advanced AWS options (#4211)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.2/clusters/creating-konvoy-cluster-aws/index.md

Select the pre-configured [AWS infrastructure provider](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-static-credentials/) (or [AWS role credentials](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-roles/)) to display the remaining options specific to AWS.

- **Region**: Select the data center region or specify a custom region.

> - **Add Infrastructure Provider Tags**: Specify tags to be set on all resources created in your infrastructure provider for this cluster. You can add up to 10 tags in this form. Different infrastructure providers have varying restrictions on their tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.

Reviewer: ```suggestion
- **Add Infrastructure Provider Tags**: Specify tags to be applied on all resources created in your infrastructure for this cluster. You can add up to 10 tags. Different infrastructure providers have varying restrictions on the usable tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.
```

> - **Configure Node Pools**: Specify pools of nodes, their size, and quantity. The default suggests a control plane pool containing 3 m5.xlarge nodes and a worker pool containing 4 m5.2xlarge nodes. You can also specify IAM instance profile, base OS, lookup format, owner ID, and AMI ID as part of each pool.

Reviewer 2: I would remove the default suggestions here as we would need to update the docs if we would update it in the code

Reviewer 2: ```suggestion
- **Configure Node Pools**: Specify pools of nodes, their machine types, and quantity. You can also specify the IAM instance profile, base OS, lookup format, owner ID, and AMI ID as part of each pool.
```

Reviewer 3: We might want to make an indented list here so we can have a line for each item. Then we can give details for the specific option.

I think we should also mention that the control-plane size has to be odd. 

The base OS, lookup format, owner ID, and AMI ID can be grouped together as they all pertain to the AMI that the nodes will use. To be accurate the options are base OS, lookup format, owner ID, **or** AMI ID. base OS, lookup format, owner ID are used to lookup the AMI, or you can specify a specific AMI with the AMI ID. If you set AMI ID we disable base OS, lookup format, owner ID in the UI.

Reviewer 4: Yeah to Rodney's point we need to make sure that you can input AMI ID OR base OS, lookup format, owner ID. Not that the software will let you.

> - **Add Infrastructure Provider Tags**: Specify tags to be set on all resources created in your infrastructure provider for this cluster. Different infrastructure providers have varying restrictions on their tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.

Reviewer 5: ```suggestion
- **Add Infrastructure Provider Tags**: Specify the resource tags created in your infrastructure provider for this cluster. Different infrastructure providers have varying restrictions on their tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.
```

Select the pre-configured [AWS infrastructure provider](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-static-credentials/) (or [AWS role credentials](../../operations/infrastructure-providers/configure-aws-infrastructure-provider-roles/)) to display the remaining options specific to AWS.

-   **Region**: Select a data center region or specify a custom region.
-   **Configure Node Pools**: Specify pools of nodes, their machine types, quantity, and the IAM instance profile.
    - **Machine Type**: Machine instance type.
    - **Quantity**: Number of nodes. The control plane must be an odd number.
    - **IAM instance profile**: Name of the IAM instance profile to assign to the machines.

> -   **AMI Image Lookup**: You can also specify the base OS, lookup format, and owner ID of the AMI Image, or the AMI ID as part of each pool. These options can be found by clicking `Show Advanced` within the Configure Node Pools section.

Reviewer 6: ```suggestion
-   **AMI Image Lookup**: You can also specify the base OS, lookup format, and owner ID of the AMI Image, or the AMI ID as part of each pool. Clicking 'Show Advanced' within the Configure Node Pools section will show these options.
```

    - **Base OS**: Base OS for Lookup search.
    - **Lookup Format**: Lookup Format string to generate AMI search name from.
    - **Owner ID**: Owner ID for AMI Lookup search.
    - **AMI ID**: AMI ID to use for all nodes.

> -   **Add Infrastructure Provider Tags**: Specify tags to be applied on all resources created in your infrastructure for this cluster. Different infrastructure providers have varying restrictions on the usable tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.

Reviewer 6: ```suggestion
-   **Add Infrastructure Provider Tags**: Specify tags applied on all resources created in your infrastructure for this cluster. Different infrastructure providers have varying restrictions on the usable tags. See the [AWS Tags User Guide][aws-tags] for more information on using tags in AWS.
```
